client/pages/ReturnBike.tsx

```diff
@@ -1,66 +1,99 @@
 import React, { useState, useEffect } from "react";
-import { Text, View, StyleSheet, FlatList} from 'react-native';
+import { Text, View, StyleSheet, FlatList, TouchableOpacity} from 'react-native';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 
 import BikeItem from '../components/BikeItem';
-import { rides } from '../data/testRides'; // switch to db/axios, use parameters
-import { bikes } from '../data/testBikes'; 
+import { getBikes } from "../services/bikes";
+import { getRides } from "../services/rides";
 
 const ReturnBike = ({navigation}) => {
     const [uid, setUid] = useState('');
     const [bike, setBike] = useState([]);
     const [ride, setRide] = useState([]);
+    const [lockCombo, setlockCombo] = useState('');
 
     // get uid from local storage
     const getUid = async () => {
         const uid = await AsyncStorage.getItem('uid');
-        console.log(uid);
         setUid(uid);
         return uid;
     };
 
     // grab current trip for the user
     const getRide = async (uid: string) => {
```

Comment:
Is this a function that would be better implemented on the backend, letting Firebase do the filtering for us? Maybe having a _getRide(uid)_ route would be nice to have in _functions_.